localstack/services/sns/sns_listener.py

```diff
@@ -414,23 +415,33 @@ def create_sns_message_body(subscriber, req_data):
         except KeyError:
             raise Exception("Unable to find 'default' key in message payload")
 
-    data = {}
-    data['MessageId'] = str(uuid.uuid4())
-    data['Type'] = req_data.get('Type', ['Notification'])[0]
-    data['Timestamp'] = timestamp(format=TIMESTAMP_FORMAT_MILLIS)
-    data['Message'] = message
-    data['TopicArn'] = subscriber['TopicArn']
+    data = {
+        'Type': req_data.get('Type', ['Notification'])[0],
+        'MessageId': str(uuid.uuid4()),
+        'Token': '',
+        'TopicArn': subscriber['TopicArn'],
+        'Message': message,
+        'SubscribeURL': '',
```

Comment:
I think this should be
```
'SubscribeURL': req_data.get('SubscribeURL', [''])[0],
```
Same for the other attributes that are hardcoded as `''` here (`Token`). Because these attributes may get passed in via `req_data`.

We should also extend the SNS test here to assert that the `SubscribeURL`/`Token` attributes are present and valid: https://github.com/localstack/localstack/blob/master/tests/integration/test_sns.py#L64-L66